nginx/subvisual.co.conf

```diff
@@ -1,5 +1,6 @@
 server {
-  listen 443; server_name subvisual.co;
+  listen 443;
+  server_name subvisual.co www.subvisual.co old.subvisual.co;
```

Comment:
Shouldn't this also be recognizing the `.info` and `.pt` domain versions (including the `www` subdomain)?